tools/cbmc/proofs/HTTP/IotHttpsClient_ReadHeader/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+set(cbmc_proof_name IotHttpsClient_ReadHeader)
+list(APPEND cbmc_test_labels ${cbmc_proof_name})
+
+set(IotHttpsClient_ReadHeader_MAX_ACCEPTED_SIZE 20 CACHE STRING "TODO what is this?")
```

Comment:
@markrtuttle please could you let me know what a better documentation string for this variable would be?

Reply:
This variable appears to be unneeded in the proof.

Reply:
done